### ISSUE TITLE
Can't open editors

### ISSUE THREAD

**Issue by dgiero** [ONLYOFFICE/Docker-DocumentServer#155]
I installed only the Docker-DocumentServer following the site instructions.
When I put the server address in my browser I reach the welcome page saying: 
**Document Server is running**

How can I start the editors? In the welcome page there isn't any link to click.

If I got it right, the DocumentServer is only a " free collaborative online office suite comprising viewers and editors for texts, spreadsheets and presentations" and one of its components is the web-apps (the editors), but how can I start them?

**Comment by ShockwaveNN** [comment]
Hi, free version of DocumentServer do not contain any DMS, so you should look out for any DMS like nextcloud or use DocumentServer Integration or Develop version, which include basic test exaple for managing documents

**Comment by dgiero** [comment]
> Hi, free version of DocumentServer do not contain any DMS, so you should look out for any DMS like nextcloud or use DocumentServer Integration or Develop version, which include basic test exaple for managing documents

So the DocumentServer by itself do nothing. They should write this somewhere.

Thanks

**Comment by ShockwaveNN** [comment]
It allow to edit documents according to our documentation in https://api.onlyoffice.com/editors/basic

**Comment by dgiero** [comment]
> This is hurting :(
I didn't mean to hurt you.
Let me rephrase:
So the DocumentServer out-of-the-box do nothing. It needs to be integrated with a website/owncloud/nextcloud. 
Is that correct?

**Comment by ShockwaveNN** [comment]
I've added some more info to readme https://github.com/ONLYOFFICE/DocumentServer/pull/538
And we think about adding some more info to main page of DocumentServer

**Comment by ShockwaveNN** [comment]
This issue was closed due to no response.
